scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -693,7 +693,12 @@ def __get_last_request_for_log( self, path ):
         return scalyr_util.seconds_since_epoch( result )
 
     def __create_log_config( self, parser, path, attributes, base_config={}, parse_as_json=False ):
-        """Convenience function to create a log_config dict from the parameters"""
+        """Convenience function to create a log_config dict
+        @param: parser - string - a parser to use if no parser is found in the attributes or base_config
```

Comment:
Call this `default_parser` as well since it is only used if there is none specified in `attributes` or `base_config`

Reply:
Done.